fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -697,6 +735,9 @@ public Response createObject(@HeaderParam(CONTENT_DISPOSITION) final ContentDisp
 
                 if (hasVersionedResourceLink(links)) {
                     resource.enableVersioning();
+                    if (resource instanceof FedoraBinary) {
+                        resource.getDescription().enableVersioning();
```

Comment:
This should be `resource.enableVersioning();`, FedoraBinary has the special enableVersioning method that enables versioning on the description as well.